### PR TITLE
Docs are not clear regarding --allow-net

### PR DESCRIPTION
**Pull request by JavaScriptDude**
As per issue found on Stack Overflow (https://stackoverflow.com/questions/61806548/deno-permissions-allow-net)

### CODE REVIEW

File: docs/getting_started/permissions.md [hunk 1]
```diff
@@ -26,12 +26,20 @@ $ deno run --allow-read=/etc https://deno.land/std/examples/cat.ts /etc/passwd
...
-This is an example to restrict host.
 
+### Network access:
+
+*fetch.ts*:
 ```ts
 const result = await fetch("https://deno.land/");
 ```
 
+This is an exemple on how to whitelist hosts/urls:
```
Comment (marcosc90) [inline docs/getting_started/permissions.md:37]:
```suggestion
This is an example on how to whitelist hosts/urls:
```

**Comment by Drunpy** [comment]
Failing tests seems to be about code format, please run the code formatting tool:
https://github.com/denoland/deno/blob/master/docs/contributing/development_tools.md#lint-and-format

**Comment by JavaScriptDude** [comment]
Thanks. Setting up testing now.

File: docs/getting_started/permissions.md [hunk 2]
```diff
@@ -27,12 +27,22 @@ $ deno run --allow-read=/etc https://deno.land/std/examples/cat.ts /etc/passwd
...
+_fetch.ts_:
 
 ```ts
 const result = await fetch("https://deno.land/");
 ```
 
+This is an example on how to whitelist hosts/urls:
+
+```shell
+$ deno run --allow-net=deno.land https://deno.land/std/examples/curl.ts https://deno.land/ fetch.ts
```
Comment (bartlomieju) [inline docs/getting_started/permissions.md:41]: @JavaScriptDude can you also include an example with comma separated list?
```
$ deno run --allow-net=deno.land,https://deno.land/std/examples/curl.ts,https://deno.land/ fetch.ts
```

Comment (JavaScriptDude) [reply docs/getting_started/permissions.md:41]: Sure thing. Just confirming on Discourse that its a stable feature.

**Comment by JavaScriptDude** [comment]
Looking at `flags.rs`, `fn resolve_hosts(paths: Vec<String>) -> Vec<String> {` is what is parsing the tokens passed to `--allow-net` and it only accepts `<domain_name[:<port>]`. I've updated the doc to reflect this.

**Comment by JavaScriptDude** [comment]
From Discourse discussion with @bartlomieju, full URLs should be supported including protocol, path and resource. A bug will be raised to discuss this. For now I propose we update the docs to reflect the current working condition as it stands so we are friendly to the new developers and update when the API changes.

**Pull request merged**